.vscode/launch.json

```diff
@@ -0,0 +1,134 @@
+{
+    // see https://github.com/microsoft/vscode-cmake-tools/blob/main/docs/debug-launch.md
+    "version": "0.2.0",
+    "configurations": [
+        // sclang target
+        {
+            "type": "lldb",
+            "request": "attach",
+            "name": "macOS: sclang attach (lldb)",
+            "program": "${workspaceFolder}/build/lang/Debug/sclang",
+            "waitFor": true,
+        },
+        {
+            "name": "macOS: sclang launch (lldb)",
```

Comment:
Is this sclang binary actually launchable from this location? I seem to remember that the class library and some other things mean that it doesn't really work?

Reply:
Well it depends on what you want to do with it - I think the class library is not available at this point, but you can actually start the CLI which may be sufficient.